homeassistant/components/apcupsd/binary_sensor.py

```diff
@@ -14,7 +14,9 @@
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up an APCUPSd Online Status binary sensor."""
-    add_entities([OnlineStatus(config, apcupsd.DATA)], True)
+    apcups_data = hass.data[apcupsd.DOMAIN]
```

Comment:
`from . import DOMAIN` and move `KEY_STATUS` and `VALUE_ONLINE` consts in this file (used only here).

Or use a `const.py` file